angular.json

```diff
@@ -150,6 +150,5 @@
         }
       }
     }
-  },
-  "defaultProject": "cloudstack-ui"
```

Comment:
leave it here

Reply:
default project field deprecated and will be removed in next major release